src/patternfly/demos/Tabs/tabs--page-grid-view.hbs

```diff
@@ -10,7 +10,7 @@
         {{#> wrapper newcontext tabs--page-grid-view--id=(concat tabs--page-grid-view--id "-subtabs")}}
           {{#> l-flex l-flex--modifier="pf-m-column"}}
             {{#> l-flex-item}}
-              {{#> tabs tabs--IsSecondary="true" tabs--id=tabs--page-grid-view--id}}
+              {{#> tabs tabs--id=tabs--page-grid-view--id}}
```

Comment:
On the [nested tabs design](https://marvelapp.com/prototype/9766ieg/screen/88398077) the callout is to add a bottom border to the secondary tabs in the card, but this change makes them primary tabs instead - which adds a border, but also increases the font-size. Was it decided that those tabs should be primary tabs? Or should they remain secondary and we just add a border? cc @mcarrano @mmenestr